src/qutip_cupy/__init__.py

```diff
@@ -5,6 +5,13 @@
 # to an official optional dependency
 import warnings
 
+try:
+    import cupy
+except ModuleNotFoundError:
+    raise RuntimeError('''qutip_cupy requires cupy to be installed, please install
+                         cupy by following the instructions at https://docs.cupy.dev/en/stable/install.html''')
```

Comment:
```suggestion
    raise RuntimeError(
        "qutip_cupy requires cupy to be installed, please install "
        "cupy by following the instructions at https://docs.cupy.dev/en/stable/install.html"
    )
```
The triple-quoted string creates a string literal with a "\n ... many spaces ..." inside it. This isn't what we want inside an error message. Two string literals one after the other are automatically concatenated by the Python parser (following what C does :).